### PR TITLE
Technical documentation for new features coming on chart 1.4.0 release

### PR DESCRIPTION
**Pull request by pchico83**

### CODE REVIEW

File: src/content/self-hosted/administration/configuration.mdx [hunk 1]
```diff
@@ -324,13 +324,14 @@ Use `tolerations.devPool` to limit the autoscaler analysis to a subset of cluste
...
 - `memory.down`: Decrease the cluster size when the Memory consumption is lesser than this value. It defaults to 50 percent.
-- `min`: Minimum number of nodes in the cluster. It defaults to 1.
-- `nodePoolLabel`: The node label that identifies the node pool of the node. For example, the value in GKE is `cloud.google.com/gke-nodepool`. In EKS the value is `eks.amazonaws.com/nodegroup`. If set, the autoscaler scales each node pool independently.
+- `nodes.increment`: The number of new nodes to request when all the current nodes are overloaded. e.g. if this value is 3, the autoscaler will request 3 new nodes when all the cluster nodes are overloaded. It defaults to 1.
+- `nodes.min`: Minimum number of nodes in the cluster. It defaults to 1.
+- `nodes.max`: Maximum number of nodes in the cluster. It defaults to 10. Zero means unlimited.
+- `nodes.poolLabel`: The node label that identifies the node pool of the node. For example, the value in GKE is `cloud.google.com/gke-nodepool`. In EKS the value is `eks.amazonaws.com/nodegroup`. If set, the autoscaler scales each node pool independently.
```
Comment (AgustinRamiroDiaz) [inline src/content/self-hosted/administration/configuration.mdx:334]: Should we announce this as breaking change in the release notes?

Comment (pchico83) [reply src/content/self-hosted/administration/configuration.mdx:334]: It's not a breaking change, the previous values also work

File: src/content/self-hosted/administration/configuration.mdx [hunk 2]
```diff
@@ -560,20 +564,31 @@ The webhook service. Ingress creation, generation of hostnames, enforcement of p
...
-### affinities
+### affinity
 
 Apply default affinities to pods deployed in namespaces created by Okteto.
 
+- `devPool`: Affinities for pods created on namespaces managed by Okteto.
+- `oktetoPool`: Affinities for pods created during the Okteto installation.
+
 ```yaml
 affinities:
```
Comment (AgustinRamiroDiaz) [inline src/content/self-hosted/administration/configuration.mdx:575]:
```suggestion
affinity:
```

File: src/content/self-hosted/administration/configuration.mdx [hunk 3]
```diff
@@ -560,20 +564,31 @@ The webhook service. Ingress creation, generation of hostnames, enforcement of p
...
           values:
           - "true"
+      weight: 10
+  oktetoPool:
+    preferredDuringSchedulingIgnoredDuringExecution:
+    - preference:
+        matchExpressions:
+        - key: dev.okteto.com/overloaded
+          operator: DoesNotExist
+      weight: 50
```
Comment (AgustinRamiroDiaz) [inline src/content/self-hosted/administration/configuration.mdx:591]: Nit: I find this indentation more readable

```
  devPool:
    preferredDuringSchedulingIgnoredDuringExecution:
      - preference:
          matchExpressions:
            - key: cloud.google.com/spot
              operator: In
              values:
                - "true"
        weight: 10
  oktetoPool:
    preferredDuringSchedulingIgnoredDuringExecution:
      - preference:
          matchExpressions:
            - key: dev.okteto.com/overloaded
              operator: DoesNotExist
        weight: 50

```

File: src/content/self-hosted/administration/configuration.mdx [hunk 4]
```diff
@@ -623,6 +638,21 @@ convertLoadBalancedServices:
   enabled: true
 ```
 
+### crds
+
+Configures the the Okteto custom resource definition installation.
+
+- `annotations`: The annotations to apply to the CRDs created during the Okteto installation.
+- `keep`: Keep CRDs on chart uninstall (defaults to `true`).
+- `install`: Install and upgrade CRDs (defaults to `true`).
```
Comment (AgustinRamiroDiaz) [inline src/content/self-hosted/administration/configuration.mdx:647]:
```suggestion
- `install`: Install and upgrade CRDs on chart install (defaults to `true`).
```

**Review by AgustinRamiroDiaz — APPROVED** [review]

**Review by AgustinRamiroDiaz — APPROVED** [review]

**Pull request merged**